### DISCUSSION TITLE
'Commit Change' button is disable when I try to edit index.html file

### DISCUSSION THREAD

**Discussion by samzea** [community/community#28958] [Other Feature Feedback, Questions, & Ideas] [answered]
I tried to edit my index.html file but Commit Change button is disable. Any help would be appreciated!

**Comment by airtower-luna** [comment, marked as answer]
The button should become enabled after you make changes. Does that not work for you?

**Reply by samzea** [reply]
Thanks for the answer. It didn't work even after I made change 😭

**Reply by airtower-luna** [reply]
Can you share a screenshot? It's hard to guess what might be going on without any details.

**Reply by samzea** [reply]
I just tried again. The button is enable now. Thanks for your help. I very appreciate it! 😀😊👍

**Reply by airtower-luna** [reply]
Glad to hear it works now! :smile_cat:
